### ISSUE TITLE
alphanet.sh not quoting client commands correctly

### ISSUE THREAD

**Issue by fredcy** [tezos/tezos#97]
Given the "identity" program from Milo's site, the following fails:
```
$ ./alphanet.sh client run program container:identity.tzs on storage Unit and input '"hello world"'
Fatal error:
  Command not found
Try `-help` for a list of options and commands.
```
If I run
```
bash -x ./alphanet.sh client run program container:progs/identity.tzs on storage Unit and input '"hello world"'
```
the output includes the following toward the end:
```
+ docker exec -it tezos-alphanet tezos client run program file:/tmp/copied/identity.tzs on storage Unit and input '"hello' 'world"'
```
The quoting around "hello world" is clearly wrong.

If I instead use an input value with no spaces the command works fine:
```
$ ./alphanet.sh client run program container:progs/identity.tzs on storage Unit and input '"hello"'
storage
  Unit
output
  "hello"
```

**Comment by tzs1** [comment]
Don't know if correlated, but trying contract example "Reverse" (https://www.michelson-lang.com/#sec-1-31), I get a fatal error: command not found.

This is my command:
`./alphanet.sh client run program container:reverse.tzs on storage Unit and input '(List "string1" "string2")`

Same error when "running" the contract deployed into the alphanet blockchain.

**Comment by yg10** [comment]
It looks like that docker is misquoting shell strings.
```
[~/tezos/tezos]> ./alphanet.sh shell
bash-4.3$ tezos-client  run program file:/tmp/copied/identity.tzs on storage Unit and input '"hello world"'
storage
  Unit
output
  "hello world"

```

**Comment by venator** [comment]
Fixed with commit b1e5c719da044d3344af2109b5e6fcb153408d31

**Comment by fredcy** [comment]
I tried again with the latest alphanet.sh just now and the problem no longer appears. So the problem is fixed (for my case, anyway).
